lib/elixir_analyzer.ex

```diff
@@ -166,7 +168,14 @@ defmodule ElixirAnalyzer do
     code_path = Path.dirname(full_code_path)
     code_file = Path.basename(full_code_path)
 
-    {code_path, code_file, exercise_config[:analyzer_module] || ElixirAnalyzer.TestSuite.Default}
+    exemplar_path =
+      case meta_config["files"]["exemplar"] do
+        [path | _] -> Path.join(params.path, path)
+        _ -> nil
```

Comment:
A few line above there was already something similar
```elixir
relative_code_path = meta_config["files"]["solution"] |> hd()
```
This ignores anything but the first file and crash if it's empty.

Reply:
I thought about this a bit more.
For the exemplar code, I guess it's mostly fine because AFAIK, there is only one file for every exercise fo far, and unexpected situations (no files, multiple files) are very unlikely. 
For student code, that is more of an issue. Not for website submissions, since I don't think they can create new files, but for CLI submissions, there could be many (Tim loves 3+ files solutions). So the analysis would miss anything outside of the main file. 

Am I correct in saying that the directory/file structure in Elixir doesn't matter as long as it's in `lib`, and that only module names are important? 
If that is the case, a simple solution is to concatenate all the files that are in `lib` (as opposed to `meta_config["files"]["solution"]` which only points to the stub files) and pretend they were all one file. Actually we can do the same for the exemplar.